python/plugins/db_manager/db_plugins/spatialite/connector.py

```diff
@@ -172,16 +172,17 @@ def getTables(self, schema=None, add_sys_tables=False):
         items = []
 
         sys_tables = ["SpatialIndex", "geom_cols_ref_sys", "geometry_columns", "geometry_columns_auth",
-                      "views_geometry_columns", "virts_geometry_columns", "spatial_ref_sys",
-                      "sqlite_sequence",  # "tableprefix_metadata", "tableprefix_rasters",
```

Comment:
Have you checked git history to see why these were commented out?

Reply:
Yep; the commented out tables were part of the original commit, 6 years ago, therefore we have no justification for those. I suspect it might have been some sort of temporary debugging state that was left behind.